frontend/components/Accounts/ui.ts

```diff
@@ -1,4 +1,6 @@
-import styled from "styled-components";
+import styled, { CSSObject } from "styled-components";
+
+// TODO: component library
```

Comment:
I'd say we don't have to add this TODO for now since we are definitely getting back to all of this!